cypress/e2e/smoke/create_tx.cy.js

```diff
@@ -35,17 +36,18 @@ describe('Queue a transaction on 1/N', () => {
 
     // Insert amount
     cy.get('input[name="amount"]').type(`${sendValue}`)
+
+    cy.contains('Next').click()
   })
 
   it('should create a queued transaction', () => {
     // Wait for /estimations response
-    cy.intercept('POST', '/**/multisig-transactions/estimations').as('EstimationRequest')
-
-    cy.contains('Next').click()
+    cy.intercept('POST', '**/multisig-transactions/estimations', {
+      statusCode: 200,
+      body: { currentNonce, recommendedNonce, safeTxGas: '45006' },
```

Comment:
Can you try reverting this and check if it works now? The backend was fixed AFAIK.

Reply:
ok sure!

Reply:
It seems that in the first execution `create_tx.cy` test passed but in the second try it failed 😔

https://cloud.cypress.io/projects/exhdra/runs/e5e81fcb-d9cf-46c2-bc7b-e93ba5275027/test-results/14d6de18-41f1-4cab-95f4-44b33584e3ca/video

Reply:
I am re-running it again 🤞

Reply:
`create_tx.cy.js` test [failed](https://cloud.cypress.io/projects/exhdra/runs/74555bdc-d4ef-4687-98ff-d1f0a0fb38a5/test-results/4a73ccbc-ab2c-486b-9e00-3c66c6fb77d5/video) again....

Reply:
Damn.